.github/workflows/ci.yml

```diff
@@ -38,3 +38,9 @@ jobs:
       run: |
         ./gradlew release -Prelease.customUsername=${{ secrets.REPOSITORY_ACCESS_TOKEN }}
         ./gradlew publish
+    - name : Test
+      env :
+        ORG_GRADLE_PROJECT_signingPassword : ${{ secrets.SIGNING_PASSWORD }}
+        ORG_GRADLE_PROJECT_signingKey : ${{ secrets.SIGNING_KEY }}
```

Comment:
Can you help me build the context behind those constants?
What's the process, how you build them(not asking for details for starting point) 

My understanding - Those are passwords to publish `db-replica` in Maven Central Repository? 
Do we do sth specific for Atlassian here?

Reply:
https://docs.gradle.org/current/userguide/signing_plugin.html